plugins/modules/ovirt_template.py

```diff
@@ -845,6 +873,49 @@ def _get_vnic_profile_mappings(module):
     return vnicProfileMappings
 
 
+def import_template(module, connection):
+    templates_service = connection.system_service().templates_service()
+    if search_by_name(templates_service, module.params['name']) is not None:
+        return False
+
+    events_service = connection.system_service().events_service()
+    last_event = events_service.list(max=1)[0]
+
+    external_template = module.params['kvm']
+    imports_service = connection.system_service().external_template_imports_service()
+    imported_template = imports_service.add(
+        otypes.ExternalTemplateImport(
+            template=otypes.Template(
+                name=module.params['name']
+            ),
+            url=external_template.get('url'),
+            cluster=otypes.Cluster(
+                name=module.params['cluster'],
+            ) if module.params['cluster'] else None,
+            storage_domain=otypes.StorageDomain(
+                name=external_template.get('storage_domain'),
+            ) if external_template.get('storage_domain') else None,
+            host=otypes.Host(
+                name=external_template.get('host'),
+            ) if external_template.get('host') else None,
+        )
```

Comment:
There was another parameter `clone` added in the API but the python SDK was not yet released with it so after the merge, I'll create an issue on it to add the parameter.
https://github.com/oVirt/ovirt-engine-api-model/blob/master/src/main/java/types/ExternalTemplateImport.java#L114-L129